make/deps.mk

```diff
@@ -77,7 +77,7 @@ dep_erlcloud = git https://github.com/erlcloud/erlcloud 3.2.4
 
 dep_erlazure = git https://github.com/lazedo/erlazure.git add-start-link
 
-dep_couchbeam = git https://github.com/2600hz/erlang-couchbeam 86d946646ad9dee1dee305b15d5c59e4c8e60cc2 # as per kazoo master branch commit SHA refered from 2600hz-kazoo5 branch
+dep_couchbeam = git https://github.com/2600hz/erlang-couchbeam 469f3b103ac332311a8878c5c1e885ff79724045 # latest SHA from couchbeam master branch
```

Comment:
We've spoken about this before, we have to be careful when changing these as we were using 2600hz and 2600hz-kazoo5 which both have multiple modifications to them that are used in the rest of the code:
https://github.com/benoitc/couchbeam/compare/master...2600hz:erlang-couchbeam:2600hz

If we change to use the upstream branch (which master is still synced with) we lose those changes and there will be dialyzer errors and almost certainly breakage at runtime.  We need to find another way around this even if it's creating our own fork to resolve.